libs/material/domain/src/model/Material.ts

```diff
@@ -0,0 +1,26 @@
+import { MaterialId } from './MaterialId';
+import { MaterialName } from './MaterialName';
+import { MaterialImage } from './MaterialImage';
+
+
+import { MaterialPrimitives } from '../utils';
+import { MaterialIsActive } from './MaterialIsActive';
+import { MaterialDeletedAt } from './MaterialDeletedAt';
+
+export class Material {
+  constructor(readonly id: MaterialId, readonly name: MaterialName, readonly image: MaterialImage, readonly isActive: MaterialIsActive, readonly deletedAt?: MaterialDeletedAt) {}
```

Comment:
Esto se podría cambiar para mejorar la legibilidad del código
```suggestion
  constructor(
    readonly id: MaterialId, 
    readonly name: MaterialName, 
    readonly image: MaterialImage, 
    readonly isActive: MaterialIsActive, 
    readonly deletedAt?: MaterialDeletedAt
  ) {}
```